client/index.css

```diff
@@ -12,6 +12,11 @@
   font-display: swap;
   src: local('Montserrat SemiBold'), local('Montserrat-SemiBold'), url(assets/fonts/montserrat-semi-bold.woff2) format('woff2');
 }
+
+#loading{
```

Comment:
can we rename this something like `#preload`?

Reply:
Ok sir , i will change it to `preload`  as it will make  more sense ☺️

Reply:
ok i got this . we can use `loader` class for `#preload div` and can then add some remaining style property likewise we can use `spin` animation of index.css file but i am wondering how we can use them without involving javascript?

May be we can render this preload div using react and can then include these styles but then the problem is still remain unsolved due to delay in render of these javascript file(s)

Reply:
what should we do sir any suggestions ?

Reply:
I think the HTML should be kept as is and the CSS should be moved from the JS file to index.html in the style tag. Try that, it should work, I tried it quickly before suggesting.

Reply:
please also remember to run `./run-qa-checks`. The CI build is not working now but it will be resumed soon.

Reply:
@nemesisdesign , Thank a lot sir for your help now i have done required changes and also ran `./run-qa-checks` which you have suggested me.
And also Everything is fine from my side could you please confirm it ☺️.